apps/frontend/src/app/5_pages/LeaderboardPointsPage/LeaderboardPointsPage.constants.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { prettyTx } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../2_molecules/AmountRenderer/AmountRenderer';
+import { translations } from '../../../locales/i18n';
+import { User } from './LeaderboardPointsPage.types';
+import { BalanceRenderer } from './components/BalanceRenderer/BalanceRenderer';
+
+export const CAMPAIGN_URL = 'https://wiki.sovryn.com/en/spice-campaign';
+
+export const MAXIMUM_USERS_TO_SHOW = 50;
+export const EXTRA_SPICE_POINTS_MULTIPLIER = 0.5;
+export const RUNES_POINTS_MULTIPLIER = 3.14;
+
+export const COLUMNS_CONFIG = (isUserPosition: boolean = false) => [
+  {
+    id: '',
+    title: '',
+    cellRenderer: (user: User) => user.id,
+    className: 'hidden lg:table-cell',
+  },
+  {
+    id: 'position',
+    title: isUserPosition
```

Comment:
This doesn't make sense.
<img width="1586" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/b4856e70-39a6-4427-bcaf-848df6a2cd5a">

Take a look at how it's handled in the POWA page and do it similarly (consult with fineline or Hyde if not sure): https://github.com/DistributedCollective/sovryn-dapp/blob/98b527d4add1eb75e1f9db7da6dc534c86e70985/apps/frontend/src/app/5_pages/LeaderboardPage/components/Leaderboard/components/BaseTable/BaseTable.constants.tsx#L22